src/biblenotify/ui/main.qml

```diff
@@ -283,10 +283,12 @@ Window {
     HomeView {
         id: homeView
         setTimeView: setTimeView
+        verseText: ""
+        verseReference: ""
 
         Component.onCompleted: {
-            // TODO: Why doesn't this work?
-            // Loader.printHello()
+            homeView.verseText = root.verse[0]
+            homeView.verseReference = root.verse[1]
```

Comment:
See the comment [here](https://github.com/BibleNotify/BibleNotifyDesktop/pull/16/files#diff-3e8b53b69c64b43736f570012160187574b8048d1cd9882a1f7d0b7fb298aeb5R12-R13).

Does setting the `root.verse` properties not work [here](https://github.com/BibleNotify/BibleNotifyDesktop/pull/16/files#diff-dd3914ea29f84f5c2559011aab9d26201d20be4aa3ba9447b1844b3230c18e5bR286-R287)?